src/fmtstr.rs

```diff
@@ -113,7 +114,7 @@ impl<'a, 'b> Formatter<'a, 'b> {
         let mut pad: usize = 0;
         if let Some(mut width) = width {
             if width > len {
-                let align = self.align();
+                let mut align = self.align();
```

Comment:
why did this become mut?

Reply:
Oops, left in from when I was mutating it.